docs/usage/machine_set.md

```diff
@@ -1,54 +1,77 @@
 # Maintaining machine replicas using machines-sets
+<!-- TOC -->
 
+- [Maintaining machine replicas using machines-sets](#maintaining-machine-replicas-using-machines-sets)
+  - [Setting up your usage environment](#setting-up-your-usage-environment)
+  - [Important :warning](#important-warning)
+  - [Creating machine-set](#creating-machine-set)
+  - [Inspect status of machine-set](#inspect-status-of-machine-set)
+  - [Health monitoring](#health-monitoring)
+  - [Delete machine-set](#delete-machine-set)
+
+<!-- /TOC -->
 ## Setting up your usage environment
 
 * Follow the [steps described here](prerequisite.md)
 
-## Important :warning: 
-- Make sure that the `kubernetes/machine-set.yaml` points to the same class name as the `kubernetes/aws-machine-class.yaml`.
-- Similarily `kubernetes/aws-machine-class.yaml` secret name and namespace should be same as that mentioned in `kubernetes/aws-secret.yaml`
+## Important :warning
+
+> Make sure that the `kubernetes/machines_sets_deployments/machine-set.yaml` points to the same class name as the `kubernetes/machine_classes/aws-machine-class.yaml`.
+
+> Similarily `kubernetes/machine_classes/aws-machine-class.yaml` secret name and namespace should be same as that mentioned in `kubernetes/secrets/aws-secret.yaml`
 
 ## Creating machine-set
 
-- Modify `kubernetes/machine-set.yaml` as per your requirement. You can modify the number of replicas to the desired number of machines. Then, create an machine-set
+- Modify `kubernetes/machines_sets_deployments/machine-set.yaml` as per your requirement. You can modify the number of replicas to the desired number of machines. Then, create an machine-set:
+
 ```bash
-$ kubectl apply -f kubernetes/machine-set.yaml
+$ kubectl apply -f kubernetes/machines_sets_deployments/machine-set.yaml
```

Comment:
I can still see occurrences of `machines_sets_`- which I guess is not aligned with actual folder-structure. Can you please change the necessary occurrences.